scripts/yaml_check.py

```diff
@@ -0,0 +1,41 @@
+import yaml
+import sys
+
+def tiered_extract(db):
+  out = []
+  for _, entry in db.items():
+    for _, values in entry.items():
+      for name, decl in values.items():
+        if decl is not None and decl != '' and '/' not in decl:
```

Comment:
I think you can use `if decl and '/' not in decl` here, you don't need to be so specific, and this describes the intent better.